FutureNova/Classes/Endpoint.swift

```diff
@@ -67,6 +75,35 @@ extension Endpoint {
         self.useCommonHeaders = useCommonHeaders
     }
 
+    /// General initializer without body
+    public init(path: String,
+                            queryItems: [URLQueryItem] = [],
+                            headers: [String: String] = [:],
+                            method: Endpoint.Method = .get,
+                            useCommonHeaders: Bool = true,
+                            useCommonPath: Bool = true,
+                            customHost: String? = nil,
+                            customScheme: String? = nil,
+                            customPort: Int? = nil) {
+        self.path = path
+        self.queryItems = queryItems
+
+        var modifiedHeaders = headers
+        if modifiedHeaders["Content-Type"] == nil {
+            // Set Content-Type to application/json so backend can identify json body
+            modifiedHeaders["Content-Type"] = "application/json"
```

Comment:
don't think we need to set the `Content-Type` if there is no body so we can just keep it as 
`self.headers = headers`